canonical-web-and-design/charmhub.io
# 1270 details page header

## Done

- Add `/details` page header

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8045/charm
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [design](https://app.zeplin.io/project/5d3f1850b938ee0dc24c60c4/screen/5e2abd08b530e498135c80b6)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1270
Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1263
